app/javascript/css/components/track-nav.css

```diff
@@ -0,0 +1,62 @@
+@import "../styles";
+
+.c-track-nav {
+    --nav-height: 40px;
+
+    @apply bg-white py-12 shadow-base;
+
+    & .container {
+        @apply flex items-center;
+    }
+
+    & .back {
+        @apply block mr-24;
+        line-height: var(--nav-height);
+        & svg {
+            height: 21px;
```

Comment:
So I've tidied up this header to remove all the abnormal sizes. It'll effect the margins between various objects and how they're displayed.

We also have something called an iconButton (rename it as you wish), which is an icon just wrapped with a clickable padded area around it. I did this so we can add mouseover effects to those things in the future (sort of material design-esque).

Reply:
Cool. So that's basically what this is (where the `.back` is the button). So we just need to understand whether these IconButtons have a consistent style I guess?